src/main/java/jalau/at18/kingoftokyo/Main.java

```diff
@@ -0,0 +1,8 @@
+package jalau.at18.kingoftokyo;
+
+import jalau.at18.kingoftokyo.view.chooseplayer.ChoosePlayersFrame;
+public class Main {
+    public static void main(String[] args) {
+        new ChoosePlayersFrame();
```

Comment:
I see @saraialvarezrosas has created a `GameFrame` class as well, is this intended to be a different frame?

Reply:
Yes, for now we think that the starting frame its going to be this one.